package.json

```diff
@@ -202,6 +202,7 @@
     "testing-library-selector": "^0.2.1"
   },
   "dependencies": {
+    "@folio/plugin-query-builder": "^1.0.100000751",
```

Comment:
Do you need this specific version or we could leave just `^1.0.1`?

Reply:
Good catch, this dependency should not be here at all. I've removed